### ISSUE TITLE
NavigationView makes the app crash with SIGABRT

### ISSUE THREAD

**Issue by MattRighetti** [exyte/PopupView#6]
I've tested the Popup with this simple View

```
import SwiftUI
import PopupView

struct ContentView: View {
    
    @State var name: String = ""
    @State var isPopupPresented = false
    
    var body: some View {
        NavigationView {
            ZStack {
                Text("Test")
                    .padding()
                    .overlay(RoundedRectangle(cornerRadius: 8).stroke().foregroundColor(.green))
                
            }.popup(isPresented: $isPopupPresented, view: {
                HStack {
                    Text("The popup")
                }
                .frame(width: 200, height: 60)
                .background(Color(red: 0.85, green: 0.8, blue: 0.95))
                .cornerRadius(30.0)
            })
            
            .navigationBarTitle("Large Title")
        }
    }
}
```
but as soon as I deploy it on the simulator the view crashes with this log
```
2020-06-11 12:07:41.661664+0200 Test[2114:73858] precondition failure: invalid input index: 4
(lldb) 
```
<img width="1548" alt="Screenshot 2020-06-11 at 12 10 05" src="https://user-images.githubusercontent.com/16304728/84373228-8faf3300-abdc-11ea-882d-735710963515.png">

**Comment by f3dm76** [comment]
Hello @MattRighetti , thank you for your report. Fixed in 0.0.4, you are most welcome to check it out. Have a great day!

**Comment by MattRighetti** [comment]
The popup is not working anymore 🤔 this code is working in v0.0.3 but not on the latest version

```
struct ContentView: View {
    
    @State var name: String = ""
    @State var isPopupPresented = false
    
    var body: some View {
        ZStack {
            VStack {
                Text("Test")
                    .padding()
                    .overlay(RoundedRectangle(cornerRadius: 8).stroke().foregroundColor(.green))
                
                Button(action: {
                    self.isPopupPresented.toggle()
                }, label: { Text("Deploy").padding() })
            }.padding()
            
        }.popup(isPresented: $isPopupPresented, type: .toast, autohideIn: 3.0, view: {
            HStack {
                Text("The popup")
            }
            .frame(width: 200, height: 60)
            .background(Color(red: 0.85, green: 0.8, blue: 0.95))
            .cornerRadius(8.0)
            .padding(.bottom, 25)
        })
    }
}
```
